pkg/scheduler/framework/interface.go

```diff
@@ -593,6 +593,9 @@ type Framework interface {
 
 	// SetPodNominator sets the PodNominator
 	SetPodNominator(nominator PodNominator)
+
+	// SetBackgroundRunner sets the BackgroundRunner
+	SetBackgroundRunner(runner BackgroundRunner)
```

Comment:
Without this call, plugins will crash when they try to invoke the `RunInBackground` method. I am unsure whether this is a problem (*the* scheduler calls it) or what to do about it.

The API could be changed so that plugins work with and without a runner, but that's more complicated.

`PodNominator` is similar: it provides functionality that the framework itself doesn't have. I had also experimented with adding a new method into the `PodNominator` interface, but that felt worse.